nsqd/client_v2.go

```diff
@@ -214,7 +215,7 @@ func (c *clientV2) Identify(data identifyDataV2) error {
 }
 
 func (c *clientV2) Stats() ClientStats {
-	c.RLock()
+	c.metaLock.RLock()
```

Comment:
P.S. this has the really nice side-effect of making `/stats` requests _really_ fast, regardless of `nsqd` load, addressing long-standing issues with `nsqadmin` timing out.